### DISCUSSION TITLE
Add friendly name of webhook

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco-CMS#15158] [Features and ideas]
I wonder if it should be possible to specify a "friendly" name of webhooks created in V13, so one know whats it is used for?

Some services have this feature, but not all.

e.g. it's an option on Bitbucket, but not on GitHub.
https://magefan.com/blog/add-webhook-in-bitbucket

or a description like Adyen: payment gateway 
https://docs.adyen.com/development-resources/webhooks/

**Comment by JeffreyPerplex** [comment]
Yes, would love that idea and I would even suggest adding a description as well... Listed my thoughts over here (https://github.com/umbraco/Umbraco-CMS/discussions/15235) and closed it when I saw this one :)

**Comment by bjarnef** [comment]
@Zeegaan this will probably not be too difficult to implement. Not sure if the name should be optional or mandatory. E.g. most other entities requires a name.

**Reply by Zeegaan** [reply]
Doesn't sound too difficult no. We're in the process of discussing if Webhooks will end up as an `IEntity`, and thus will get a Name property 👍

**Comment by Zeegaan** [comment]
Implemented in https://github.com/umbraco/Umbraco-CMS/pull/18217
